package.json

```diff
@@ -38,6 +38,7 @@
     "db:migrate:prod": "sequelize db:migrate --env production",
     "db:dev": "run-s db:create:dev db:migrate:dev",
     "db:test": "run-s db:create:test db:migrate:test",
+		"sqs:init": "aws --endpoint-url=http://localhost:9602 sqs create-queue --queue-name webhooksqs",
```

Comment:
should probably use AWS CLI package for this, and make it part of the start command as well.